lib/Test2/Harness/Collector.pm

```diff
@@ -214,16 +218,49 @@ sub process_tasks {
     return $count;
 }
 
+sub send_backed_up {
+    my $self = shift;
+    return if $self->{+BACKED_UP}++;
+
+    # This is an unlikely code path. If we're here, it means the last loop couldn't process any results.
+    my $e = $self->_harness_event(0, undef, time, info => [{details => <<"    EOT", tag => "INTERNAL", debug => 1, important => 1}]);
+*** THIS IS NOT FATAL ***
+
+ * The collector has reached the maximum number of concurrent jobs to process.
+ * Testing will continue, but some tests may be running or even complete before they are rendered.
+ * All tests and events will eventually be displayed, and your final results will not be effected.
+
+Set a higher --max-open-jobs collector setting to prevent this problem in the
```

Comment:
In our case, this doesn't help at all. The issue is that the number of events is too much to process and it is doing it sequentially. The collector just can't keep up. with even a single loop of processing. Incressing `--max-open-jobs` Will not make this better and will probably make it worse.